lib/streamlit/runtime.py

```diff
@@ -0,0 +1,591 @@
+# Copyright 2018-2022 Streamlit Inc.
```

Comment:
The contents of `runtime.py` are mostly just pulled straight out of `server.py`, but I'll highlight a few changes.